tests/test_runlib.py

```diff
@@ -445,6 +445,30 @@ def test_hash_artifact_passing_algorithm(self):
     self.assertTrue("sha256" in list(_hash_artifact("foo", ["sha256"]).keys()))
 
 
+class TestLinkCmdExecTimeoutSetting(unittest.TestCase):
+  """Tests LINK_CMD_EXEC_TIMEOUT setting in settings.py file. """
+
+  def test_timeout_setting(self):
+    # Save the old timeout and make sure it's saved properly
+    timeout_old = in_toto.settings.LINK_CMD_EXEC_TIMEOUT
+
+    # Modify timeout
+    in_toto.settings.LINK_CMD_EXEC_TIMEOUT = 0.1
```

Comment:
The `int()` in `execute_link` will chop off the digits after the decimal point. Maybe we should use `float()` instead?

Reply:
Why did you resolve this comment? Do you not think that we should cast to `float` rather than to `int`?

Reply:
Oops sorry, I had corrected it locally and didn't push changes